dev/mocks.json

```diff
@@ -42,6 +43,9 @@
   "animation.json",
   "annotations-autorange.json",
   "annotations.json",
+  "automargin-push-x-extra.json",
```

Comment:
where are all these from?

Reply:
plotly.js repo

Reply:
Maybe sometime I'll set up a thing to pull the list automatically, we really don't need to keep this file in the archive - it's just a directory listing essentially.
https://github.com/plotly/plotly.js/tree/master/test/image/mocks